forge-game/src/main/java/forge/game/Game.java

```diff
@@ -880,8 +882,6 @@ public void onPlayerLost(Player p) {
                         }
                         triggerList.put(c.getZone().getZoneType(), null, c);
                         getAction().ceaseToExist(c, false);
-                        // CR 603.2f owner of trigger source lost game
```

Comment:
What about the delayed trigger? Should they still happen or not?

Reply:
they shouldn't be cleared like that
`TriggerHandler.onPlayerLost` will take care of which ones are invalid